sim/druid/item_sets_pve.go

```diff
@@ -485,12 +490,80 @@ var ItemSetFuryOfStormrage = core.NewItemSet(core.ItemSet{
 	Bonuses: map[int32]core.ApplyEffect{
 		// Swipe(Bear) also causes your Maul to hit 1 additional target for the next 6 sec.
 		2: func(agent core.Agent) {
+			druid := agent.(DruidAgent).GetDruid()
+
+			if druid.Env.GetNumTargets() == 1 {
+				return
+			}
+
+			var curDmg float64
+
+			cleaveHit := druid.RegisterSpell(Bear, core.SpellConfig{
+				ActionID:    core.ActionID{SpellID: 467217},
+				SpellSchool: core.SpellSchoolPhysical,
+				ProcMask:    core.ProcMaskEmpty,
+				Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagNoOnCastComplete | core.SpellFlagPassiveSpell,
+
+				DamageMultiplier: 1,
+				ThreatMultiplier: 1,
+
+				ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+					spell.CalcAndDealDamage(sim, target, curDmg, spell.OutcomeAlwaysHit)
+				},
+			})
+
+			cleaveAura := druid.RegisterAura(core.Aura{
+				Label:    "2P Cleave Buff",
+				Duration: time.Second * 6,
+				OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+					if result.Landed() && (spell.SpellCode == SpellCode_DruidSwipeBear) {
+						curDmg = result.Damage / result.ResistanceMultiplier
+						cleaveHit.Cast(sim, druid.Env.NextTargetUnit(result.Target))
+						cleaveHit.SpellMetrics[result.Target.UnitIndex].Casts--
+					}
+				},
+			})
+
+			core.MakePermanent(druid.RegisterAura(core.Aura{
+				Label: "S03 - Item - T2 - Druid - Guardian 2P Bonus",
+				OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+					if result.Landed() && spell.SpellCode == SpellCode_DruidMaul {
+						cleaveAura.Activate(sim)
+						curDmg = result.Damage / result.ResistanceMultiplier
+						cleaveHit.Cast(sim, druid.Env.NextTargetUnit(result.Target))
+						cleaveHit.SpellMetrics[result.Target.UnitIndex].Casts--
+					}
+				},
```

Comment:
If I'm reading the effect properly, this `OnSpellHitDealt` should check for a swipe hit and activate the cleave aura. The cleave aura is the one that should check for maul and cast a second maul if the sim has at least 2 targets